.vscode/tasks.json

```diff
@@ -116,29 +109,98 @@
             "problemMatcher": "$msCompile"
         },
         {
-            "label": "Run mgcb-editor",
+            "label": "Clean MonoGame Framework (Android)",
```

Comment:
I don't think there is a reason to add clean task 😅

Reply:
Probably my ignorance with the SDK style workflow, but I found that changes in the Content pipeline would not cause a rebuild on a framework build on Net Core 3.1(this may have been fixed since then).  

It ran with the old code and gave the message:
>> All projects are up-to-date for restore.

This is the reason for the `[DependsOn]` and `clean`.  

I had considered a VS style "rebuild" task and hiding the cleans. But AFIK, the task system does not allow purely compound tasks.

Other than the additional space on the task list, I do not see any reason not to add it.

People are free to modify these files as they see fit.  It is easier to customize via deletion than addition.

Reply:
Conntent.Pipeline references the MonoGame.Framework, therefore if you modify Content.Pipeline, there is no reason for MonoGame.Framework to be rebuilt.

> People are free to modify these files as they see fit. It is easier to customize via deletion than addition.

True, but I don't think we should keep anything except build tasks in there, otherwise you get a question of, should we keep rebuild, pack, publish, etc.

Reply:
Normally, I would block comment out the offending code to allow it to be re-added by the user, but the json format does not allow that.

For the above reasons, my PR stands **as is**, notwithstanding corrections to the`mgcb-editor` file names and locations.

The true :sweat_smile: is that the original and subsequent PR's did not take other OS's into account at all. I recall you beating that into me with my first PR. (**Thank You**, I have not forgotten that lesson.)

Reply:
> Normally, I would block comment out the offending code to allow it to be re-added by the user, but the json format does not allow that.

json format does not, but VS Codes json configuration files do allow it.